erreser.go

```
@@ -0,0 +1,630 @@
+		return
+	}
+	for _, feed := range feeds {
+		log.Println("Checking feed: ", feed.Title)
+		fp := gofeed.NewParser()
+		feedData, err := fp.ParseURL(feed.Feed)
+		if err != nil {
+			log.Println("Error parsing feed: ", err)
+			return
+		}
+		items := feedData.Items[:9]
+		updateItems(db, items, &feed)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)